macros/schema_tests/equality.sql

```diff
@@ -18,7 +18,7 @@
 
 
 
--- core SQL
+{% if target.type == 'bigquery' %}
```

Comment:
The only difference between BQ and the other adapters is `except distinct`, right? Can we make a cross-platform implementation of `except` that returns `except distinct` for bigquery and `except` for everything else?

Reply:
this one's kinda tricky, along with talking about date/time casts, we should talk about this one too. :)